### DISCUSSION TITLE
Eww shadows around widgets

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [hyprwm/Hyprland#3003] [General]
For me adding shadow via css to the most parent widget in eww results in a shadow that gets cut off by the area eww occupies. This is with ` shadow_range = 60 ` , the windows all have proper shadows but eww doesn't.
With Css Shadow:
![swappy-20230814_112824](https://github.com/hyprwm/Hyprland/assets/61395246/e6611634-7095-4ece-8ff6-6cf0003e7159)
Without Css Shadow:
![swappy-20230814_112854](https://github.com/hyprwm/Hyprland/assets/61395246/237dc897-3136-4084-a955-be5903c2569b)

**Comment by B21q** [comment]
Managed to solve?

**Comment by end-4** [comment]
have you tried adding margins
